Sources/MockDataQueue.swift

```diff
@@ -14,48 +14,48 @@
 import Foundation
 
 /// MockDataQueue - see also AEPServices/Mocks
-public class MockDataQueue: DataQueue {
+open class MockDataQueue: DataQueue {
     // swiftlint:disable identifier_name
-    let queue = ThreadSafeArray<DataEntity>()
+    public let queue = ThreadSafeArray<DataEntity>()
 
     public init() {}
 
-    public func add(dataEntity: DataEntity) -> Bool {
+    open func add(dataEntity: DataEntity) -> Bool {
```

Comment:
I would not recommend keeping all these mocks and other utils open where possible, in order to reduce the amount of code overlap and simplify the debugging experience. My understanding is that if additional support or modifications are required, contributors would participate to the development of this repository, or for custom implementations a separate mock can be created implementing all the APIs from the defined protocol.